doc/news/changes/minor/20200318DavidWells

```diff
@@ -0,0 +1,6 @@
+Improved: IBTK::CCPoissonHypreLevelSolver::solveSystem() and
+IBTK::SCPoissonSolverManager::solveSystem() now both return <code>true</code> in
```

Comment:
```suggestion
IBTK::SCPoissonSolver::solveSystem() now both return <code>true</code> in
```

Reply:
Both should be `PoissonHypreLevelSolver`.